src/extension/openFileAtLocation.ts

```diff
@@ -50,7 +50,7 @@ import * as Q from "q";
         return remoteExtension.openFileAtLocation(filename, lineNumber);
     }).done(() => { }, (reason) => {
         throw ErrorHelper.getNestedError(reason, InternalErrorCode.CommandFailed,
-            "Unable to communicate with VSCode. Please make sure it is open in the appropriate workspace.");
+            {argument: "Unable to communicate with VSCode. Please make sure it is open in the appropriate workspace.", isPii: false});
```

Comment:
Do we really need to pass error message as argument?

Reply:
In this case yes, because it is a part of error message: `"Error while exectuting command: '{0}'"`